src/extensions/RhinoAssessmentHierarchyExtension.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace DNADesign\Rhino\Extensions;
+
+use DNADesign\Rhino\Pagetypes\RhinoAccount;
+use DNADesign\Rhino\Pagetypes\RhinoCapability;
+use DNADesign\Rhino\Pagetypes\RhinoCategory;
+use DNADesign\Rhino\Pagetypes\RhinoModule;
+use SilverStripe\ORM\DataExtension;
+
+class RhinoAssessmentHierarchyExtension extends DataExtension
+{
+
+	/**
+	 * Helper methods
+	 */
+	public function getModule()
+	{
+		return $this->owner->getParentOfType(RhinoModule::class);
+	}
+
+	public function getCapability()
+	{
+		return $this->owner->getParentOfType(RhinoCapability::class);
+	}
+
+	public function getCategory()
+	{
+		return $this->owner->getParentOfType(RhinoCategory::class);
+	}
+
+	public function getAccount()
+	{
+		return $this->owner->getParentOfType(RhinoAccount::class);
+	}
+
+	public function getParentOfType($type)
+	{
+		$parents = $this->owner->getBreadcrumbItems();
+		$parents->remove($this);
```

Comment:
@alex-dna Remove($this->owner)? Just not sure if that context is needed for this operation.

Reply:
Not sure what you mean. This code is on an extension, not sure how to call that method other than on the owner...

Reply:
Yeah calling the method from the owner is correct - but the passed param is the extension. Should remove() take the owner rather? 
i.e. $parents->remove($this->owner)

Reply:
Oh, sorry I wasn't ready the right line. You are absolutely right.